src/util/ukComplianceUtils.ts

```diff
@@ -1,25 +1,14 @@
 import { LocaleStringKey } from '../locales/en_US'
 import { lstrings } from '../locales/strings'
 
-type CompliantStringKeys =
-  | 'buy_1s'
-  | 'buy_1s_quote'
-  | 'sell_1s'
-  | 'stake_earn_1s'
-  | 'stake_earn_button_label'
-  | 'stake_x_to_earn_y'
-  | 'insufficient_fees_2s'
-  | 'insufficient_fees_3s'
+type CompliantStringKeys = 'stake_earn_1s' | 'stake_earn_button_label' | 'stake_x_to_earn_y' | 'insufficient_fees_2s' | 'insufficient_fees_3s'
```

Comment:
If you add a `//` somewhere in here, you can force it back onto multiple lines so it's readable.